app/main.py

```diff
@@ -1,8 +1,26 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
```

Comment:
I can still see no annotation. Add it, if you forgot to do it) Everything else is ok

Reply:
I did not forget about it)))) please see my above reply to your proposal of the annotation. I am totally confused how to make it correctly and without errors. I think option "people: [dict[str]] = {}" was the best: all worked, tests passed, no flake8 errors in spite of required 2 arguments for dict annotation.

Reply:
"Simple is better than complex"